### PR TITLE
documentation: readme file update

### PR DESCRIPTION
**Pull request by adriansuhov**
readme file was updated with resource requirements for Hyper-V run

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +33,13 @@ LISAv2 includes below test suite categories:
 * kvp_client64
 * nc.exe
 
+4. For running Azure tests, you must have a valid Windows Azure Subscription.
+
+5. For running Hyper-V tests, the resource requirements are:
+- Hyper-V role enabled
+- At least 8 GB of memory on the Host - Most of lisav2 tests will create and start Virtual Machines (Guests) with 3.5 GB of memory assigned
+- 1 External vSwitch in Hyper-V Manager/Virtual Switch Manager. This vSwitch will be named 'External' and must have an internet connection. For Hyper-V NETWORK tests you need 2 more vSwitch types created: Internal and Private. These 2 vSwitches will have the naming also 'Internal' and 'Private'.
```
Comment (LiliDeng) [inline README.md]: We can run current code without this external switch with mandatory name "External". I tested it with rename the external switch.

            $InterfaceAliasWithInternet = (Get-NetIPConfiguration -ComputerName $HyperVHost | Where-Object {$_.NetProfile.Name -ne 'Unidentified network'}).InterfaceAlias
            $VMSwitches = Get-VMSwitch -ComputerName $HyperVHost | Where-Object {$InterfaceAliasWithInternet -match $_.Name} | Select-Object -First 1

Comment (adriansuhov) [reply README.md]: It's for Hyper-V network area testing. In the XML the switch name is External. For all other test areas, yes, External name is not mandatory

File: README.md [hunk 2]
```diff
@@ -18,13 +18,11 @@ LISAv2 includes below test suite categories:
 
 ### Prerequisite
 
-1. You must have a Windows Machine with PowerShell (v5.0 and above) as test driver.
+1. You must have a Windows Machine (Host) with PowerShell (v5.0 and above) as test driver. It should be Windows Server for localhost, or any Windows system including Windows 10 for remote host access case.
```
[the remark this answers is not in the document]
Comment (adriansuhov) [reply README.md]: I put below that Hyper-V role should be enabled for running Hyper-V tests. Isn't that enough?

File: README.md (hunk 1 above)
Comment (LiliDeng) [reply README.md]: @adriansuhov Thanks for reply.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (LiliDeng) [reply README.md]: @adriansuhov It is fine, thanks for reply

**Pull request merged**